st3/sublime_lib/_util/collection_util.py

```diff
@@ -0,0 +1,64 @@
+from collections.abc import Mapping, Sequence
+
+
+__all__ = ['projection', 'get_selector', 'isiterable', 'ismapping', 'is_sequence_not_str']
+
+
+def projection(d, keys):
+    """
+    Return a new :class:`dict` with keys restricted to values in `keys`.
```

Comment:
should mention `d` here, e.g. ```keys of ``d`` ```.

Also ``` ``keys`` ```.

```suggestion
    Return a new :class:`dict` with keys of ``d`` restricted to values in ``keys``.
```

Reply:
Done.